src/components/TimeLine/time-line.js

```diff
@@ -117,4 +122,51 @@ const processData = (data, colors) => {
   return dataSplitYears;
 };
 
+const processKtas = (ktas) => {
+
+  if (!ktas || ktas === []) return [];
+
+  let ktasSplitYears = [];
+  for (let ktaKey in ktas) {
+    let startDate = parseInt(ktas[ktaKey].start_date);
+    let endDate = parseInt(ktas[ktaKey].end_date);
+    if (isNaN(startDate) || startDate === "") startDate = 1900;
+    if (isNaN(endDate) || endDate === "") endDate = startDate;
+    if(startDate > 2000){
+      for (let targ in ktas[ktaKey].targetgroups){
```

Comment:
there's really no reason for using confusing short forms, IDEs will auto-complete anyway and its much more readable